src/mqueryfront/src/DatabaseTopology.js

```diff
@@ -7,8 +7,12 @@ import { API_URL } from "./config";
 
 class DatasetRow extends Component {
     render() {
-        return (
-            <tr>
+        return [
+            <tr
+                data-toggle="collapse"
+                data-target={"#collapsed_" + this.props.id}
```

Comment:
This should **probably** be done with refs (yeah, my crusade against ids again), but after taking a 15s look at it I think I don't think it's worth it (more problems than it's worth right now).